frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -208,30 +209,27 @@ export default memo(function HeatMap({
     return result;
   }, [selectedGeneExpressionSummariesByTissueName, geneNameToIndex]);
 
-  const initialDisplayedCellTypes = useMemo(
-    () =>
-      Object.entries(tissuesByName).reduce((acc, [_, { id }]) => {
-        if (
-          (filteredTissues.length > 0 && filteredTissues.includes(id)) ||
-          filteredTissues.length === 0
-        ) {
-          acc.add(id + id);
-        }
-        return acc;
-      }, new Set<string>()),
-    [tissuesByName, filteredTissues]
-  );
+  const initialDisplayedCellTypeIds = useMemo(() => {
+    return Object.values(tissuesByName).reduce((acc, { id }) => {
```

Comment:
use `Object.values` instead of `Object.entries`, since we don't use the keys anyway